CMakeLists.txt

```diff
@@ -93,7 +101,12 @@ if(CMAKE_INSTALL_PREFIX)
   endif()
 endif()
 
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DUJIT_OS=${UJIT_OS}")
+
 option(UJIT_ENABLE_GDBJIT "Support for DWARF data for assembled traces" ON)
+if(NOT UJIT_OS STREQUAL "UJIT_OS_LINUX")
+  set(UJIT_ENABLE_GDBJIT OFF)
+endif()
```

Comment:
Here and everywhere else in this file I'd better use this pattern:

```cmake
if(UJIT_OS STREQUAL "UJIT_OS_LINUX")
  option(UJIT_ENABLE_GDBJIT "Support for DWARF data for assembled traces" ON)
else()
  set(UJIT_ENABLE_GDBJIT OFF)
endif()
```

It's better to not give user and option to make something ON or OFF if it's not supported on this platform anyway.

Reply:
Agree.